dipy/segment/quickbundles.py

```diff
@@ -4,135 +4,136 @@
 from dipy.tracking.distances import bundles_distances_mdf
 from warnings import warn
 
-warn(DeprecationWarning("Module 'dipy.segment.quickbundles' is deprecated,"
-			" use module 'dipy.segment.clustering' instead"))
+deprecation_msg = ("Class 'dipy.segment.quickbundles.QuickBundles' is"
+                   " deprecated, instead use module "
+                   " 'dipy.segment.clustering.QuickBundles'.")
+warn(DeprecationWarning(deprecation_msg))
 
 
 class QuickBundles(object):
-    
-    def __init__(self,tracks,dist_thr=4.,pts=12):
-        """ Highly efficient trajectory clustering 
-        
+
+    def __init__(self, tracks, dist_thr=4., pts=12):
+        """ Highly efficient trajectory clustering
+
         Parameters
-        -----------
-        tracks : sequence of (N,3) ... (M,3) arrays,
-                    trajectories (or tractography or streamlines)
-                    
-        dist_thr : float, 
-                    distance threshold in the space of the tracks
-        pts : int, 
-                number of points for simplifying the tracks 
-                       
+        ----------
+        tracks : sequence of (N,3) ... (M,3) arrays
+            trajectories (or tractography or streamlines)
+        dist_thr : float
+            distance threshold in the space of the tracks
+        pts : int
+            number of points for simplifying the tracks
+
         Methods
-        --------
+        -------
         clustering() returns a dict holding with the clustering result
         virtuals() gives the virtuals (track centroids) of the clusters
-        exemplars() gives the exemplars (track medoids) of the clusters        
-        
-        Citation
-        ---------
-        
-        E.Garyfallidis, "Towards an accurate brain tractography", PhD thesis, 2012 
-        
+        exemplars() gives the exemplars (track medoids) of the clusters
+
+        References
+        ----------
+        E.Garyfallidis, "Towards an accurate brain tractography",
```

Comment:
While we're at it, can we reformat this to a sphinx reference? Also, I believe we can refer to the Quickbundles paper (http://www.ncbi.nlm.nih.gov/pmc/articles/PMC3518823/) here, rather than to Eleftherios's thesis.